main/glossary/README.md

```diff
@@ -67,151 +102,362 @@ synchronously compared for structural equivalence.
+- `afterDeadline`: The offer is automatically cancelled after a deadline, as determined by its timer and deadline properties.
+
 ## Facet
 
-A particular view or API of an object. An object can have many facets.
+A particular view or API of an object. An object can have many facets. Two Agoric uses are:
+- Deposit Facet: A facet of a [purse](#purse). Anyone with a reference to its deposit facet object can add 
+  appropriately branded assets to the purse, but cannot withdraw assets from the purse or find out its balance.
+- Public Facet: A set of methods and properties for an object that a developer chooses to be publicly visible and usable.
 
 ## Fungible
 A fungible asset is one where all exemplars of the asset are interchangeable. For example, if you 
 have 100 one dollar bills and need to pay someone five dollars, it does not matter which
 five one dollar bills you use. Also see [non-fungible](#non-fungible).
 
 ## Handle
-A handle is a unique identifier implemented as a JavaScript object. Only its identity is meaningful, so handles do not have properties. Unlike number or string identifiers, handles are unforgeable. This means the only way to know a handle identity is being given an object reference, and no identity can be guessed and no fake identity will succeed. 
-
-For example, Zoe often uses `offerHandle` to refer to offers. Zoe contracts can use an offer's `offerHandle` as the key for requesting the current allocation of an offer or reallocating the offer's assets.
+A handle is a unique identifier implemented as a JavaScript object. Only its identity 
+is meaningful, so handles do not have properties. Unlike number or string identifiers, 
+handles are unforgeable. This means the only way to know a handle identity is being given 
+an object reference, and no identity can be guessed and no fake identity will succeed. 
+
+## Harden
+A hardened object’s properties cannot be changed, so the only way to interact with a hardened object is through its methods.
+`harden()` is similar to `Object.freeze()` but more powerful. For more about `harden()`, see
+its [section in the JavaScript Distributed Programming Guide](/distributed-programming.md#harden)
+
+## IBC
+The Inter-Blockchain Communication protocol, used to by blockchains to communicate with each other. A short article about IBC
+is available [here](https://www.computerweekly.com/blog/Open-Source-Insider/What-developers-need-to-know-about-inter-blockchain-communication).
+
+## Invitation
+To participate in a contract instance, one must hold an invitation to do so. Contracts often 
+return a creator invitation on their instantiation, in case the contract instantiator wants 
+to immediately participate. Otherwise, the contract instance must create any additional invitations. 
+These, or any invitation held by a party, can be distributed via any means the holder wishes. An
+invitation could be emailed directly to a friend, posted on a bulletin board, etc. When you receive 
```

Comment:
hmm, well, we don't have any way of emailing ERTP payments to a friend yet, so I think we shouldn't say that. The invitation can only be distributed one way: by sending it to someone else's wallet.

Reply:
I thought you could email an object reference? Now have "Otherwise, the contract instance must create any additional invitations. These, or any invitation held by a party, are distributed by sending it to someone's wallet. When you receive 
an invitation, you should validate it via the [InvitationIssuer](#invitationissuer)."

But if the only way is to send it to a wallet, which claims/validates by depositing it in a purse, when does anyone use the invitationIssuer to do validation of an invitation?